test/README.md

```diff
@@ -8,78 +8,77 @@ These instructions will get you run the integration tests on altinn-app-frontend
-
 ### Running tests against a remote environment
 
 1. Start your local development server for `app-frontend-react`:
 
 ```cmd
-    yarn start
+yarn start
 ```
 
 2. Run the tests against a remote environment:
 
 ```cmd
-    yarn run cy:test:all -e environment=tt02
+npx cypress run --env environment=tt02 -s 'test/e2e/integration/*/*.ts'
 ```
 
-Other remote environments could also be used (see `e2e/config/*.json`).
-
 ### Running tests locally
 
-1. Clone the apps (
-   [ttd/frontend-test](https://dev.altinn.studio/repos/ttd/frontend-test),
-   [ttd/anonymous-stateless-app](https://dev.altinn.studio/repos/ttd/anonymous-stateless-app),
-   [ttd/stateless-app](https://dev.altinn.studio/repos/ttd/stateless-app),
-   [ttd/signing-test](https://dev.altinn.studio/repos/ttd/signing-test), and
-   [ttd/expression-validation-test](https://dev.altinn.studio/repos/ttd/expression-validation-test)
-   ) to be tested.
+1. Clone [app-localtest](https://github.com/Altinn/app-localtest) and follow the [instructions in the README](https://github.com/Altinn/app-localtest/blob/main/README.md) to start the local environment.
+
+2. Clone one or more of the apps we've made automatic tests for:
 
-2. Create (or update) `.env` file with the correct paths (see `template.env`)
+- [ttd/frontend-test](https://dev.altinn.studio/repos/ttd/frontend-test)
+- [ttd/anonymous-stateless-app](https://dev.altinn.studio/repos/ttd/anonymous-stateless-app)
+- [ttd/stateless-app](https://dev.altinn.studio/repos/ttd/stateless-app)
+- [ttd/signing-test](https://dev.altinn.studio/repos/ttd/signing-test)
+- [ttd/expression-validation-test](https://dev.altinn.studio/repos/ttd/expression-validation-test)
 
-3. To start localtest, app frontend, and the app you configured above, run the command below.
-   (Skip to step 4 if the solutions are already running). The command will not finish, but it
-   will start the app-frontend server in development mode. This command may take some time,
-   depending on if LocalTest has been setup earlier or not, and if the docker cache is hit or not.
-   When the output of this command seems to have stopped, you can continue to the next step.
+3. Start the app you want to test:
 
 ```cmd
-    yarn run cy:before:appfrontend
+cd <app-folder>/App
+dotnet run
 ```
 
-4. Start the tests for a given app from a new terminal:
+4. Start the tests for a given app:
+
+If you ran app-localtest with Docker:
 
 ```cmd
-    yarn run cy:test:frontend -e environment=local
+npx cypress run --env environment=docker -s 'test/e2e/integration/frontend-test/*.ts'
 ```
 
-or
+If you ran app-localtest with podman:
 
 ```cmd
-    yarn run cy:test:stateless -e environment=local
+npx cypress run --env environment=podman -s 'test/e2e/integration/frontend-test/*.ts'
 ```
 
-or
+5. Stop the running app (using `Ctrl+C` or `Cmd+C` in the terminal where it's running) and
```

Comment:
It's `Ctrl+C` on Mac as well, not `Cmd`

Reply:
Haha, good point. I should have known that.. :facepalm: